srcpkgs/lsof/template

```diff
@@ -12,12 +16,13 @@ license="custom:lsof"
 homepage="https://github.com/lsof-org/lsof"
 changelog="https://raw.githubusercontent.com/lsof-org/lsof/master/00DIST"
 distfiles="https://github.com/lsof-org/lsof/releases/download/${version}/lsof-${version}.tar.gz"
-checksum=2f8efa62cdf8715348b8f76bf32abf59f109a1441df35c686d23dccdeed34d99
+checksum=180e6284aff184d94d273e34f7264edc2af849c07b1c5d6a4183d4d402734245
 
 post_extract() {
 	# Fix GitHub CI quirks
-	if [ $XBPS_BUILD_ENVIRONMENT = void-packages-ci ]; then
-		patch -Np1 <"$FILESDIR/fix-github-ci.patch"
+	if [ "$XBPS_BUILD_ENVIRONMENT" = void-packages-ci ]; then
+		lnum="$(sed -n '/if (gethostname(hnm, sizeof(hnm) - 1)) {/=' tests/LTsock.c)"
+		sed "/^@@.*@@$/s/###/$lnum/g" "$FILESDIR/fix-github-ci.patch" | patch -Np1
```

Comment:
templates should not be modifying files outside of the build dir. it would be better to just rm this test in CI

Reply:
Sorry, that was a completely stupid and overcomplicated attempt at solving the problem..

... the patch itself was outdated.

I just updated it.